synapse/http/site.py

```diff
@@ -139,8 +147,8 @@ def requester(self, value: Union[Requester, str]) -> None:
         # If there's no authenticated entity, it was the requester.
         self.logcontext.request.authenticated_entity = authenticated_entity or requester
 
-    def get_request_id(self):
-        return "%s-%i" % (self.get_method(), self.request_seq)
+    def get_request_id(self) -> str:
+        return f"{self._instance_name}/{self.get_method()}-{self.request_seq}"
```

Comment:
Is this the same thing that winds up in the logs?
I can see this perhaps making things a bit noisier, but not sure.